lemur/certificates/service.py

```diff
@@ -459,14 +459,15 @@ def create(**kwargs):
     """
     Creates a new certificate.
     """
-    # Validate destinations do not overlap accounts
+    # Validate destinations do not overlap accounts for the same plugin
     if "destinations" in kwargs:
-        dest_accounts = {}
+        dest_plugin_accounts = {}
         for dest in kwargs["destinations"]:
+            plugin_accounts = dest_plugin_accounts.setdefault(dest.plugin_name, {})
```

Comment:
Optional style suggestion: other code in Lemur uses `defaultdict(dict)` to achieve similar behavior in a slightly more concise way.

Reply:
Didn't know about defaultdict, nice. I'll leave as-is, setdefault pattern is fairly common too.